specification/security/resource-manager/readme.md

```diff
@@ -89,6 +89,9 @@ suppressions:
   - code: AvoidAdditionalProperties
     from: Microsoft.Security\preview\2024-03-01\securityConnectors.json
     reason: This is a property used across all API versions. changing it would be a breaking change, and is required for 
+  - code: InvalidFormat
```

Comment:
Have you filed a bug for this?

Reply:
I don't see "byte" as a valid format for strings here..
https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-7.3

so where is it coming from?

Reply:
I'll remove this suppression since it already passed before (mdeOnboardings.json original version)